metrics/metrics_test.go

```diff
@@ -524,10 +532,10 @@ func TestStartMetricsEndPoint_Noop(t *testing.T) {
 		csi_sidecar_operations_seconds_bucket{driver_name="fake.csi.driver.io",grpc_status_code="OK",method_name="/csi.v1.Controller/ControllerGetCapabilities",le="+Inf"} 1
 		csi_sidecar_operations_seconds_sum{driver_name="fake.csi.driver.io",grpc_status_code="OK",method_name="/csi.v1.Controller/ControllerGetCapabilities"} 20
 		csi_sidecar_operations_seconds_count{driver_name="fake.csi.driver.io",grpc_status_code="OK",method_name="/csi.v1.Controller/ControllerGetCapabilities"} 1
-`
+	`
 
 	actualMetrics := string(contentBytes)
-	if err := VerifyMetricsMatch(expectedMetrics, actualMetrics, ""); err != nil {
+	if err := VerifyMetricsMatch(expectedMetrics, actualMetrics, ProcessStartTimeMetric); err != nil {
```

Comment:
Do we have a unit test that does validate this new metric is output?

Reply:
Added